10_1_roads.js

```diff
@@ -0,0 +1,21 @@
+import { buildGraph } from './10_1_graph.js';
+
+const roads = [
```

Comment:
You shouldn't use splitting in such cases, because e.g. the name of the house could contain "-", like "James-Dean's House".

It's better in this case to do array of arrays of strings:
[
     ["Alice's House","Bob's House"],
     ["Alice's House","Cabin"],
...
]

Best way would be to declare places as a,b,c,d and so on, and describe what is what in a comment - program doesn't care about name, it's just a variable on a graph.
[
[a,b]
[a,c]
[b,d]
]
etc.